src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/OnboardingSupervisor.kt

```diff
@@ -131,11 +137,19 @@ internal class OnboardingSupervisor(
         }
     }
 
+    private fun retrieveSkipEvmSwapVenues() {
```

Comment:
we can alternatively do this by making it async and calling it in each routing call to make sure the swap venues info is most up to date. it feels a little excessive to do that but im open to opinions here